nextflow.config

```diff
@@ -112,10 +111,18 @@ params {
 }
 
 // Load base.config by default for all pipelines
-includeConfig 'conf/base.config'
+try {
+  includeConfig 'conf/base.config'
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/base.config")
```

Comment:
Note sure about that

Reply:
ok, yeah I had problems were the error messsage for failing to load configs was very confusing. But maybe this is also something to deal with upstream and not in sarek

Reply:
no worry, we'll fix that